.env

```
@@ -30,10 +30,9 @@ ESPRESSO_SEQUENCER1_API_PORT=24001
 ESPRESSO_SEQUENCER2_API_PORT=24002
 ESPRESSO_SEQUENCER3_API_PORT=24003
 ESPRESSO_SEQUENCER4_API_PORT=24004
-ESPRESSO_SEQUENCER_MAX_BLOCK_SIZE=1mb
-ESPRESSO_SEQUENCER_BASE_FEE=1
 ESPRESSO_SEQUENCER_URL=http://sequencer0:${ESPRESSO_SEQUENCER_API_PORT}
 ESPRESSO_SEQUENCER_STORAGE_PATH=/store/sequencer
+ESPRESSO_SEQUENCER_GENESIS_FILE=/genesis
```

```suggestion
ESPRESSO_SEQUENCER_GENESIS_FILE=/genesis.toml
```
Not important, just looks strange to me without extension.